source/includes/_clients.md

```diff
@@ -60,15 +69,25 @@ Update an existing client. Only supply values you want to update. The rest will
 use BrightLocal\Api;
 
 $api = new Api('<INSERT_API_KEY>', '<INSERT_API_SECRET>');
-$success = $api->put('v1/clients-and-locations/clients/', [
+$success = $api->put('v1/clients-and-locations/clients/', array(
```

Comment:
Let's show short array syntax [] rather than array(). It's supported in all but very old versions of PHP.